pixeltable/store.py

```diff
@@ -427,6 +427,34 @@ def delete_rows(
         status = conn.execute(stmt)
         return status.rowcount
 
+    def dump_rows(self, version: int, filter_view: StoreBase, filter_view_version: int) -> Iterator[dict[str, Any]]:
```

Comment:
Missing method comment.

Also, I would have expected the filter_view to be optional (what about base tables?).